src/components/button-toggle/button-toggle.scss

```diff
@@ -29,9 +29,44 @@
   }
 }
 
+.carbon-button-toggle--grouped {
+  .carbon-button-toggle__input:checked ~ .carbon-button-toggle__label{
+    border-color: $blue-sky;
+  }
+
+  + .carbon-button-toggle--grouped {
+    margin-left: -1px;
+  }
+
+  label {
+    border-radius: 0;
+  }
+
+  &:first-of-type label {
+    border-radius: 4px 0 0 4px;
+  }
+
+  &:last-of-type label {
+    border-radius: 0 4px 4px 0;
+  }
+
+  &--disabled {
+    border: 1px solid #D8D9DC;
+  }
+}
+
+.carbon-button-toggle--small {
+  label {
+    padding: 5px 8px 5px 8px;
+    font-weight: 700;
+    font-size: 12px;
+  }
+}
+
 .carbon-button-toggle__label--disabled,
 .carbon-button-toggle__label--disabled:hover {
   background-color: $carbon-button__background-color--disabled !important;
+  border: 1px solid $carbon-button__background-color--disabled !important;
```

Comment:
is this required, or added by mistake?

Reply:
This was needed, the blue border was showing on focus when disabled

Reply:
Can you just do `border-color` instead?